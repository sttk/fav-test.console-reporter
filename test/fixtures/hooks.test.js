'use strict';

describe('hooks', function() {

  before(function() {
    // runs before all tests in this block
    console.log('runs before all tests in this block');
  });

  after(function() {
    // runs after all tests in this block
    console.log('runs after all tests in this block');
  });

  beforeEach(function() {
    // runs before each test in this block
    console.log('runs before each test in this block');
  });

  afterEach(function() {
    // runs after each test in this block
    console.log('runs after each test in this block');
  });

  // test cases
  it('test case 1', function() {
  });
  it('test case 2', function() {
  });
});

