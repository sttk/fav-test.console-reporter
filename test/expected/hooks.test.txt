

  hooks
runs before all tests in this block
runs before each test in this block
    ✓ test case 1
runs after each test in this block
runs before each test in this block
    ✓ test case 2
runs after each test in this block
runs after all tests in this block


  2 passing (3ms)

